bot/cogs/snakes.py

```diff
@@ -2,10 +2,33 @@
 import logging
 from typing import Any, Dict
 
+from googleapiclient.discovery import build
 from discord.ext.commands import AutoShardedBot, Context, command
+from discord import Embed
+
+import aiohttp
+import json
+import async_timeout
+import random
+import difflib
 
 log = logging.getLogger(__name__)
 
+# Probably should move these somewhere
+BASEURL = "https://en.wikipedia.org/w/api.php?format=json&action=query&prop=extracts|pageimages&exintro=&explaintext=&titles={}&redirects=1"
+PYTHON = {
+    "name": "Python",
+    "info": """Python is a species of programming language, \
+commonly used by coding beginners and experts alike. It was first discovered \
+in 1989 by Guido van Rossum in the Netherlands, and was released to the wild \
+two years later. Its use of dynamic typing is one of many distinct features, \
+alongside significant whitespace, heavy emphasis on readability, and, above all, \
+absolutely pain-free software distribution... *sigh*""",
+    "image": "https://www.python.org/static/community_logos/python-logo-master-v3-TM-flattened.png"
+}
+with open("bot/snakes.txt") as file:
+    SNAKES = list(map(lambda ln: ln.strip('\n'), file.readlines()))
```

Comment:
I don't even know where to begin with this line.

- don't use map and lambdas, use list comps. this isn't Python 1.0.
- a simple `ln.strip()` will also remove newlines (and spaces) from the start and end. why even specify the `\n`?
- don't call the file variable `file`, that's a python 2 builtin which means syntax highlighting looks funky in most editors.
- `ln` is a shitty variable name. just call it `line`. this isn't code golf.